src/main/java/net/dries007/tfc/client/screen/CreateTFCWorldScreen.java

```diff
@@ -106,6 +113,9 @@ protected void init()
         options.addBig(
             continentalness = pctOption("tfc.create_world.continentalness", settings.continentalness())
         );
+        options.addBig( // min max match 1.18 range comment
+            rockLayerScale = scaleOption("tfc.create_world.rocklayerscale", 0, 32, settings.rockLayerScale())
+        );
```

Comment:
The comment here is useless, this ideally should be paired with a biome scale option as well.

Reply:
If we're no longer doing zooms via integer scale, this comment is worthless. Wanted to give some motivation for the magic numbers. 

biome scale as in forest/lake/marsh, or terrain features?